invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides

```diff
@@ -107,3 +107,29 @@ code {
 .generate-margins(30, left, l);
 .generate-margins(30, bottom, b);
 .generate-margins(30, top, t);
+
+.generate-truncate-classes(@max, @i: 1) when (@i =< @max){
+  .truncate-@{i} {
+    display: -webkit-box !important;
+    -webkit-line-clamp: @i;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+  }
+  .generate-truncate-classes(@max, (@i + 1));
+}
+
+.generate-truncate-classes(5);
+
+
+/* Truncate css  */ 
+
+.truncate-container {
+  
+  p{
```

Comment:
can we avoid to harcode a `p` element? maybe we use a `div` to avoid the margin?